project/GrasshopperBuild.scala

```diff
@@ -117,7 +117,6 @@ object GrasshopperBuild extends Build {
     .configs( IntegrationTest )
     .settings(buildSettings: _*)
     .settings(
-      Revolver.settings ++
```

Comment:
Thanks, I meant to add a comment about this.  `Revolver` adds the `~re-start` feature to `sbt`, which listens for code changes, and re-deploys the apps.   The `client` module is just a lib, and has no "main" method, so `Revolver` doesn't know what to do with it, and breaks `re-start` when running it for the whole `grasshopper` project.